src/test/java/com/github/mc1arke/sonarqube/plugin/ce/pullrequest/github/v4/GraphqlCheckRunProviderTest.java

```diff
@@ -195,12 +195,33 @@ public void createCheckRunHappyPathErrorStatus() throws IOException, GeneralSecu
     }
 
 
+
     private void createCheckRunHappyPath(QualityGate.Status status) throws IOException, GeneralSecurityException {
+        String[] messageInput = {
+            "issue 1",
+            "issue 2",
+            "issue 3",
+            "issue 4",
+            "issue 5",
+            "issue 6",
+            "issue \\ \" \\\" \"\\ \\\\ \"\" 7"
+        };
+
+        String[] messageOutput ={
```

Comment:
Formatting: could you ad a space between the `=` and the `{` please?

Reply:
This is fixed now